src/SFA.DAS.Campaign.UnitTests/Infrastructure/Api/Queries/WhenGettingALandingPage.cs

```diff
@@ -17,49 +16,111 @@ namespace SFA.DAS.Campaign.UnitTests.Infrastructure.Api.Queries
+        public async Task AndForcePreviewIsFalse_AndAllowPreviewIsTrue_AndPreviewIsNotWanted_ThenTheLandingPageIsReturnedFromTheProductionAPI(
+            bool allowPreview, bool previewWanted, GetLandingPageQuery query, Page<LandingPage> response, [Frozen] Mock<IOptions<CampaignConfiguration>> config, [Frozen] Mock<IApiClient> client, GetLandingPageQueryHandler handler)
+        {
+            query.Preview = previewWanted;
+            SetupMockConfig(config, allowPreview, false);
+            client.Setup(o => o.Get<Page<LandingPage>>(It.Is<GetLandingPageRequest>(r => r.GetUrl == $"landingpage/{query.Hub}/{query.Slug}"))).ReturnsAsync(response);
+
+            var actual = await handler.Handle(query, CancellationToken.None);
+
+            Assert.Multiple(() =>
+            {
+                client.Verify(o => o.Get<Page<LandingPage>>(It.Is<GetLandingPageRequest>(r => r.GetUrl == $"landingpage/{query.Hub}/{query.Slug}")), Times.Once);
+                client.Verify(o => o.Get<Page<LandingPage>>(It.IsAny<GetLandingPagePreviewRequest>()), Times.Never);
+                actual.Should().NotBeNull();
+                actual.Page.Should().NotBeNull();
+            });
+        }
+
+        [Test, MoqInlineAutoData(false, false)]
+        public async Task AndForcePreviewIsFalse_AndAllowPreviewIsFalse_AndPreviewIsNotWanted_ThenTheLandingPageIsReturnedFromTheProductionAPI(
+            bool allowPreview, bool previewWanted, GetLandingPageQuery query, Page<LandingPage> response, [Frozen] Mock<IOptions<CampaignConfiguration>> config, [Frozen] Mock<IApiClient> client, GetLandingPageQueryHandler handler)
+        {
+            query.Preview = previewWanted;
+            SetupMockConfig(config, allowPreview, false);
+            client.Setup(o => o.Get<Page<LandingPage>>(It.Is<GetLandingPageRequest>(r => r.GetUrl == $"landingpage/{query.Hub}/{query.Slug}"))).ReturnsAsync(response);
+
+            var actual = await handler.Handle(query, CancellationToken.None);
+
+            Assert.Multiple(() =>
+            {
+                client.Verify(o => o.Get<Page<LandingPage>>(It.Is<GetLandingPageRequest>(r => r.GetUrl == $"landingpage/{query.Hub}/{query.Slug}")), Times.Once);
+                client.Verify(o => o.Get<Page<LandingPage>>(It.IsAny<GetLandingPagePreviewRequest>()), Times.Never);
+                actual.Should().NotBeNull();
+                actual.Page.Should().NotBeNull();
+            });
         }
 
         [Test]
-        [RecursiveMoqInlineAutoData(true)]
-        [RecursiveMoqInlineAutoData(false)]
-        public async Task Then_The_Api_Is_Called_With_The_Valid_Request_Parameters_Then_The_Landing_Page_Is_Returned_From_The_Api_With_The_Menu(
-            bool preview, GetLandingPageQuery query, Page<LandingPage> response, [Frozen] Mock<IApiClient> client, [Frozen] Mock<IOptions<CampaignConfiguration>> config, GetLandingPageQueryHandler handler)
+        [RecursiveMoqInlineAutoData(true, false)]
```

Comment:
If a parameter is always the same then it should not be a parameter it should be fixed and the name of the test will say what it is